lib/auth/auth.js

```diff
@@ -114,8 +114,6 @@ auth.generateV4Headers =
         }
         const payloadChecksum = crypto.createHash('sha256').update(payload)
                                       .digest('hex');
-        const path = request.path;
```

Comment:
I'm not sure what these changes have to do with the matrix but I just made the same changes in a separate PR on Arsenal so if you rebase this should go away from this PR.